mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -1254,4 +1254,157 @@ Feature: Requests tests
     And request requestEntityRequest
     And retry until responseStatus == 422
     When method POST
-    And match $.errors[0].message == patronBlockCondition.blockMessage
\ No newline at end of file
+    And match $.errors[0].message == patronBlockCondition.blockMessage
+
+  Scenario: Test request sorting by service point name, shelving order
+    * def holdingsRecordId1 = call uuid1
+    * def callNumber1 = 'FAT5356CN2'
+    * def callNumber2 = 'FAT5356CN1'
+    * def servicePointId1 = call uuid1
+    * def servicePointName1 = 'SPN2'
+    * def servicePointCode1 = 'SPC2'
+
+    * def servicePointId2 = call uuid1
+    * def servicePointName2 = 'SPN1'
+    * def servicePointCode2 = 'SPC1'
+
+    * def servicePointEntityRequest1 = read('samples/service-point-entity-request.json')
+    * servicePointEntityRequest1.id = servicePointId1
+    * servicePointEntityRequest1.name = servicePointName1
+    * servicePointEntityRequest1.code = servicePointCode1
+
+    Given path 'service-points'
+    And request servicePointEntityRequest1
+    When method POST
+    Then status 201
+
+    * def servicePointEntityRequest2 = read('samples/service-point-entity-request.json')
+    * servicePointEntityRequest2.id = servicePointId2
+    * servicePointEntityRequest2.name = servicePointName2
+    * servicePointEntityRequest2.code = servicePointCode2
+
+    Given path 'service-points'
+    And request servicePointEntityRequest2
+    When method POST
+
+    # post an owner
+    * def ownerId = call uuid1
+    * def ownerEntityRequest = read('samples/feefine/owner-entity-request.json')
+    * ownerEntityRequest.id = ownerId
+    Given path 'owners'
+    And request ownerEntityRequest
+    When method POST
+    Then status 201
+
+    # post a material type
+    * def materialTypeId1 = call uuid1
+    * def materialTypeId2 = call uuid1
+    * def materialTypeName1 = 'Book'
+    * def materialTypeName2 = 'Text'
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostMaterialType') { extMaterialTypeId: #(materialTypeId1), extMaterialTypeName: #(materialTypeName1) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostMaterialType') { extMaterialTypeId: #(materialTypeId2), extMaterialTypeName: #(materialTypeName2) }
```

Comment:
Is it really necessary to create materiel types? There may be default ones, like 'book'. 
Also, do we actually need two different material types for this test?